salesforce/bulkwrite.go

```diff
@@ -52,10 +52,15 @@ type BulkWriteParams struct {
 	Mode BulkWriteMode
 }
 
+type BulkWriteParams BulkJobParams
+
 type BulkWriteMode string
 
 // BulkWriteResult is what's returned from writing data via the BulkWrite call.
-type BulkWriteResult struct {
+type BulkWriteResult BulkJobResult
```

Comment:
same as above, return result is the same for bulk write and bulk delete

Reply:
same as above, let's keep it as BulkWriteResult

Reply:
I think keeping it BulkWriteXX will confuse others. I think we can use BulkOperationXX. Salesforce distinguishes those jobs by field "operation"

Reply:
like operation==delete, operation==upsert, etc.

Reply:
yep that sounds good